rpcserver.go

```diff
@@ -2327,11 +2342,18 @@ func (s *rpcServer) prepareLeasesResponse(ctx context.Context,
 					clearingPrice = batch.ClearingPrices[duration]
 				}
 
+				premiumAmt := chanAmt
+				auctionType := ourOrder.Details().AuctionType
+				if auctionType == order.BTCOutboundLiquidity {
+					premiumAmt += btcutil.Amount(
+						selfChanBalance,
```

Comment:
In most cases though, we expect the channel amount to be nothing, and the actual value being the self channel balance right?

Reply:
Exactly, for the `BTCOutboundLiquidity`the chan amount needs to be always 100k (1 unit). This way we can roll it out and check the demand for this feature without having to make big refactor